src/odin/maneuversbuilder.cc

```diff
@@ -2322,6 +2342,130 @@ void ManeuversBuilder::EnhanceSignlessInterchnages(std::list<Maneuver>& maneuver
   }
 }
 
+uint16_t ManeuversBuilder::GetExpectedTurnLaneDirection(Maneuver& maneuver) const {
```

Comment:
logic to determine which turn lane to expect based on the maneuver - will continue to iterate/refactor as needed